src/content/docs/en/reference/experimental-flags/csp.mdx

```diff
@@ -1,5 +1,5 @@
 ---
-title: Experimental Content Security Policy
+title: Experimental Content Security Policy (CSP)
```

Comment:
The reason I added `CSP` is that it helps with the search. I searched CSP and couldn't find the page cc @sarah11918